lib/sass/calcite-web/components/_dropdown.scss

```diff
@@ -12,46 +12,69 @@
   }
 }
 
+%dot {
+  &:before {
+    content: "•";
+    color: $gray;
+    position: absolute;
+    left: 10px;
+    opacity: 0;
+  }
+  &:hover, &:focus {
+    background-color: $off-white;
+    color: $darker-gray;
+    text-decoration: none;
+    &:before {
+      opacity: 1;
+    }
+  }
+  &.is-active {
+    font-weight: 600;
+    &:before {
+      opacity: 1;
+      color: $blue;
+    }
+  }
+}
+
   @mixin dropdown-link() {
-    position: relative;
-    display: block;
     @include box-sizing(border-box);
-    padding: $baseline/3;
     @include font-size(-2);
+    position: relative;
+    padding: $baseline/2.5 $baseline/1;
+    display: block;
     color: $darker-gray;
-    background-color: $white;
-    border-bottom: none;
+    font-weight: inherit;
     text-align: left;
-    border-left: none;
-    border-right: none;
-    border-top: 1px solid $lightest-gray;
-    white-space: nowrap;
     cursor: pointer;
-    width: 100%;
-    &:hover {
-      background-color: $off-white;
-      text-decoration: none;
-      color: $darker-gray;
-    }
-    &.is-active, &:focus {
-      text-indent: -3px;
-      border-left: 3px solid $blue;
-    }
+    background-color: transparent;
+    overflow: visible;
+    @extend %dot;
+
     @if ($include-right-to-left) {
       html[dir="rtl"] & {
         text-align: right;
       }
-      html[dir="rtl"] &.is-active, html[dir="rtl"] &:focus {
-        border-left: none;
-        border-right: 3px solid $blue;
+      html[dir="rtl"] &.is-active:before, html[dir="rtl"] &:focus:before, html[dir="rtl"] &:hover:before {
+        right: 10px;
+        padding-right: 0;
       }
     }
   }
 
   @mixin dropdown-title() {
     @include dropdown-link();
-    background-color: $off-white;
+    font-weight: 600;
     cursor: auto;
+    border-bottom: 1px solid $lightest-gray;
+    margin: $baseline/3 $baseline/1.5 0 $baseline/1.5;
```

Comment:
if we make the margin-bottom on this `-1px` it helps alleviate this weird UI moment when you hover on the first link under a title:

<img width="201" alt="Screen Shot 2019-03-20 at 2 45 38 PM" src="https://user-images.githubusercontent.com/1031758/54721338-f4392a80-4b1e-11e9-9845-cc809193d3ff.png">

Reply:
here it is with -1px, the hover blends in so it feels more seamless:

<img width="202" alt="Screen Shot 2019-03-20 at 2 47 01 PM" src="https://user-images.githubusercontent.com/1031758/54721379-1cc12480-4b1f-11e9-9b6c-df7d0ca2a47a.png">

Reply:
Behind door number 3.... Making the first-child link have a transparent 1px border that on hover becomes the color of the partial-width header bottom. That way the border beneath the headline is maintained but there is no staggering.

Couple other styling things: 

Could we maybe tighten up the space between items by reducing y-padding 2-3 px ish? I think that would put it more in line with what's on DevExt now.

I think the active link and the header text are good candidates for `500` weight.

Cool with no animation / fade in or out for hover state?

Reply:
Will work on these

Reply:
updated spacing on title and links. Also added a fade in transition `$transition`